templates/Makefile

```diff
@@ -1,3 +1,8 @@
+init:
+	go mod init <% index .Files `Repository` %>
+	go mod edit -replace project/database=./database
+	go mod edit -require project/database@v1.0.0
```

Comment:
this will yield a go.mod that looks like, I couldn't figure out any other way to do local packages, what is the best way to do so? 
```
module github.com/davidcheung/backend-service08182

go 1.14

replace project/database => ./database

require project/database v1.0.0

```

Reply:
Hmm why would you need to do this? You should be able to import `project/database` normally without this..

Reply:
I believe I'd need the prefix of my `github.com/davidcheung/backend-service08182` to import it otherwise

Reply:
I worry a little that this could lead to issues with other modules or confusion down the road since we're not doing things in the traditional way, but we can give it a shot.
In general I think whenever there's a decision between making the templates easier / cleaner or the rendered code easier / cleaner, we should opt for clean rendered code.

Reply:
okay 👍  yeah I was trying to minimize the footprint of templating so the code template stays more code like, but I guess it's better to have more logic in the template to ensure it's easy to understand